portfolio/src/main/webapp/style.css

```diff
@@ -2,16 +2,52 @@ html {
   font-size: 100%;
 }
 
+body {
+  background-image: url(resources/bgTile.png);
+  background-repeat: repeat;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+#content p {
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+}
+
 #greeting-container {
   margin-top: 1.25em;
 }
 
+#home {
+  margin-top: 1.25em;
+  margin-bottom: 1.25em;
+  text-align: center;
+}
+
+#left {
+  float: left;
+}
```

Comment:
I definitely recommend looking into [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) when you have some time! It's sort of the newer school way to lay out elements and may help with your projects down the line.

Reply:
Ok, I'll look into it!